docs: Update README.md

iton0 opened the pull request:
Updated Lazy.nvim Installion Configuration

iton0 committed:
docs: Update README.md

Updated Lazy.nvim Installion Configuration

comfysage commented:
pulling the options out of the `opts` field seems unnecessary.
the `config` field should always be prefered over the `init` field, since `config` can be lazy loaded. e.g., when other colorschemes are installed. this is also why i don't like putting `vim.cmd.colorscheme('evergarden')` in the plugin spec.
the priority field seems like a good idea tho (as suggested [here](https://github.com/folke/lazy.nvim/blob/8f19915175395680808de529e4220da8dafc0759/doc/lazy.nvim.txt#L248)).

please read [this](https://github.com/folke/lazy.nvim/blob/8f19915175395680808de529e4220da8dafc0759/doc/lazy.nvim.txt#L106). hopefully it will help you with your nvim configuration <3

thnx for the contribution <3

iton0 commented:
Thank you for taking the time to explain. One area I am still having trouble with is the vim.cmd.colorscheme('evergarden') because if I dont have it in the config it will default the the default Neovim colorscheme.

I also changed the commit to just include the priority spec.

iton0 committed:
docs: Update README.md

- Updated Lazy.nvim Installation config

Added the priority spec such that the colorscheme is loaded first.

comfysage commented:
> One area I am still having trouble with is the vim.cmd.colorscheme('evergarden') because if I dont have it in the config it will default the the default Neovim colorscheme.

setting the colorscheme should probably be run somewhere else in your neovim config based on your setup or personal preference.
